docs/common/craft-parts/reference/plugins/ruby_plugin.rst

```diff
@@ -0,0 +1,128 @@
+.. _craft_parts_ruby_plugin:
+
+Ruby plugin
+===========
+
+The Ruby plugin manages Ruby gems and the Ruby interpreter. It can build and
+bundle different variants of the interpreter, build gems with tools like rake or
+Bundler, and install gems.
+
+.. _craft_parts_ruby_plugin-keywords:
+
+Keys
+----
+
+This plugin provides the following unique keys.
+
+
+ruby-gems
+~~~~~~~~~
+
+**Type:** list of strings
+
+The gems to install.
+
+
+ruby-use-bundler
+~~~~~~~~~~~~~~~~
+
+**Type:** boolean
+
+**Default:** False
+
+Whether to use Bundler to build the gems.
+
+
+ruby-flavor
+~~~~~~~~~~~
+
+**Type:** string
+
+**Default:** None
+
+The Ruby interpreter to build and include. The supported interpreters are:
+
+- ``ruby``
+- ``jruby``
+- ``truffleruby``
+- ``mruby``
+
+
+ruby-version
+~~~~~~~~~~~~
+
+**Type:** string
+
+**Default:** None
+
+The version of the Ruby interpreter to build.
+
+
+ruby-shared
+~~~~~~~~~~~
+
+**Type:** boolean
+
+**Default:** False
+
+When set to ``true``, the plugin builds ``libruby.so``, a shared library that
+other binaries can link against.
+
+
+ruby-use-jemalloc
+~~~~~~~~~~~~~~~~~
+
+**Type:** boolean
+
+**Default:** False
+
+Whether to build Ruby with support for jemalloc.
+
+
+ruby-configure-options
+~~~~~~~~~~~~~~~~~~~~~~
+
+**Type:** list of strings
+
+Extra arguments to pass to the ``configure`` script when building the Ruby
+interpreter.
+
+
+.. _ruby-details-begin:
+
+Dependencies
+------------
+
+The Ruby plugin needs the ``ruby`` interpreter to run Ruby programs but does not
```

Comment:
> `ruby` interpreter

Nitpick: do we really need to specify the binary name here and in the next paragraph? Doesn't "Ruby interpreter" mean the same thing?